src/python/director/viewerclient.py

```diff
@@ -96,10 +96,17 @@ def serialize(self):
         }
 
 
-class Triad(BaseGeometry, namedtuple("Triad", [])):
+class Triad(BaseGeometry):
+    __slots__ = ["tube", "scale"]
+    def __init__(self, scale=1.0, tube=False):
```

Comment:
I'd probably do that by adding `tube_radius` as an additional parameter and then deprecating `tube` eventually. That's a good idea.